tests/unit/test_service_steps.py

```diff
@@ -1158,3 +1161,92 @@ def test_eks_call_step_creation():
         },
         'End': True
     }
+
+
+@patch.object(boto3.session.Session, 'region_name', 'us-east-1')
+def test_step_functions_start_execution_step_creation():
```

Comment:
test name should include the integration pattern you're testing or "default" if you make it optional (as it is in the CDK)